adhocracy-plus/templates/account/signup.html

```diff
@@ -52,7 +52,28 @@ <h1>{% trans "Register" %}</h1>
             <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}"/>
         {% endif %}
 
-        {% include 'a4_candy_contrib/includes/form_field.html' with field=form.captcha %}
+        {% with field=form.captcha %}
+        <div class="form-group">
+            <label for="{{ field.id_for_label }}">
+                {{ field.label }}{% if field.field.required %}<span role="presentation" title="{% trans 'This field is required' %}">*</span>{% endif %}
```

Comment:
@phillimorland we did copy the wrong formfield template. But with this one (https://github.com/liqd/adhocracy-plus/blob/main/apps/contrib/templates/a4_candy_contrib/includes/form_field.html), the label is also there. :)

Is there another place where we also need to add the tabindex to the label? And if yes, could you point me to it?

Reply:
also will fix and add to other form